generator/routing_helpers.hpp

```diff
@@ -31,4 +37,6 @@ bool ParseOsmIdToFeatureIdMapping(std::string const & osmIdsToFeatureIdPath,
                                   std::map<osm::Id, uint32_t> & osmIdToFeatureId);
 bool ParseFeatureIdToOsmIdMapping(std::string const & osmIdsToFeatureIdPath,
                                   std::map<uint32_t, osm::Id> & featureIdToOsmId);
+std::pair<MwmSet::MwmId, MwmSet::RegResult> RegisterSingleMwm(
```

Comment:
Please, move this to generator common helpers, not to quite specific routing helpers.

Reply:
I guess we don't have generator common helpers, do we? I propose to rename generator/routing_helpers to generator/helpers. What do you think about it?

Reply:
I moved SingleMwmIndex to generator/utils